doc/setup/SETUP.md

```diff
@@ -63,7 +63,7 @@ If you use a password, you need to adapt it in Artemis/gradle/liquibase.gradle.
 
 **Please note:** Artemis uses Spring profiles to segregate parts of the application configuration and make it only available in certain environments. For development purposes, the following program arguments can be used to enable the `dev` profile and the profiles for JIRA, Bitbucket and Bamboo:
 
-    --spring.profiles.active=dev,bamboo,bitbucket,jira,artemis
+    --spring.profiles.active=dev,bamboo,bitbucket,jira,artemis,automaticText
```

Comment:
Can you please add some documentation (e.g. in a new README file) where the python server code is stored, how the corresponding service is setup and so on?

Please also update the hardware software mapping deployment diagram

Reply:
A link to a README in another open Github Repo might be fine as well